### PR TITLE
[OSCD] Bad Opsec Sacrificial Processes Argument Discrepancy

### PR DESCRIPTION
**Pull request by invrep-de**
[OSCD] Bad Opsec Sacrificial Processes Argument Discrepancy

### CODE REVIEW

File: rules/windows/process_creation/win_bad_opsec_sacrificial_processes.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+title: Bad Opsec Defaults Sacrificial Processes With Improper Arguments
+id: a7c3d773-caef-227e-a7e7-c2f13c622329
+status: experimental
+description: Detects attackers using tooling with bad opsec defaults e.g. spawning a sacrificial process to inject a capability into the process without taking into account how the process is normally run, one trivial example of this is using rundll32.exe without arguments as a sacrificial process (default in CS, now highlighted by c2lint), running WerFault without arguments (Kraken - credit @am0nsec), and other examples.
```
Comment (yugoslavskiy) [inline rules/windows/process_creation/win_bad_opsec_sacrificial_processes.yml:4]: I believe as you are using the `@` sign in the field, you need to put quotes at the beginning and the end of the `description` field

File: rules/windows/process_creation/win_bad_opsec_sacrificial_processes.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    - attack.t1085      # legacy
+    - attack.t1218.011
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection:
+        CommandLine:
+            - '*\WerFault.exe'
+            - '*\rundll32.exe'
```
Comment (yugoslavskiy) [inline rules/windows/process_creation/win_bad_opsec_sacrificial_processes.yml:21]: Could you please provide an example of the command you would like to detect with this logic?

Comment (invrep-de) [reply rules/windows/process_creation/win_bad_opsec_sacrificial_processes.yml:21]: Sure. This detects attempts to run any of the two commands *without* arguments, which is how these are typically used as part of sacrificial processes in adversarial activity, but NOT how they are typically used in benign scenarios (see references).

File: rules/windows/process_creation/win_bad_opsec_sacrificial_processes.yml (hunk 1 above)
Comment (invrep-de) [reply rules/windows/process_creation/win_bad_opsec_sacrificial_processes.yml:4]: Thanks. Let me fix it.

File: rules/windows/process_creation/win_bad_opsec_sacrificial_processes.yml (hunk 2 above)
Comment (yugoslavskiy) [reply rules/windows/process_creation/win_bad_opsec_sacrificial_processes.yml:21]: Thanks, @invrep-de! Then it makes sense to change it to `ProcessName` and use `endswith` value modifier without `*`.

Comment (invrep-de) [reply rules/windows/process_creation/win_bad_opsec_sacrificial_processes.yml:21]: Yes, great point! Let me fix it.

Comment (invrep-de) [reply rules/windows/process_creation/win_bad_opsec_sacrificial_processes.yml]: Fixed. Thanks.

File: rules/windows/process_creation/win_bad_opsec_sacrificial_processes.yml (hunk 1 above)
Comment (invrep-de) [reply rules/windows/process_creation/win_bad_opsec_sacrificial_processes.yml]: Fixed.

File: rules/windows/process_creation/win_bad_opsec_sacrificial_processes.yml (hunk 2 above)
Comment (yugoslavskiy) [reply rules/windows/process_creation/win_bad_opsec_sacrificial_processes.yml]: Well, it's still not ProcessName, but according to the description — it's fine. Execution without any command line arguments would look exactly this way. Well done!

File: rules/windows/process_creation/win_bad_opsec_sacrificial_processes.yml (hunk 1 above)
Comment (yugoslavskiy) [reply rules/windows/process_creation/win_bad_opsec_sacrificial_processes.yml]: Thanks!

**Review by yugoslavskiy — APPROVED** [review]
@thomaspatzke @Neo23x0 guys, I believe that this PR could be merged (:
please let us know what do you think about it if there is anything you would like to change.

**Pull request merged**